doc-content/enterprise-only/using-ansible-rhdm-in-ONAP/rules-engines-and-rhdm-con.adoc

```diff
@@ -0,0 +1,21 @@
+[id='rules-engines-and-rhdm-con']
+= Rules Engines and {PRODUCT}
+
+A rules engine is a software element that makes decisions based on business logic built using declarative format. Rules engine can be very basic providing support for simple rules to very complicated based on sophisticated algorithms.
+
+{PRODUCT} provides a versatile set of decision capabilities, including a rules engine based on the link:https://access.redhat.com/documentation/en-us/red_hat_jboss_bpm_suite/6.2/html/development_guide/rule_evaluation_with_phreak_algorithm[PHREAK] algorithm (an evolution of the well know Rete algorithm), a link:https://www.omg.org/spec/DMN/About-DMN/[DMN v1.1 FEEL Compliance Level 3] runtime, support for link:https://www.packtpub.com/mapt/book/networking_and_servers/9781783288625/7/ch07lvl1sec47/pmml[PMML], as well as a Complex Event Processing engine.
+
+The PHREAK reasoning algorithm provides a runtime that easily scales to hundreds of thousands of rules in a single rules execution environment, while providing low-latency and high-performance business rules execution.
+
+Being a lightweight engine, {PRODUCT} can be used as the decision service and runtime in various architectures and deployment topologies, including, but not limited, embedded systems and microservices architectures. The decision engine can be combined with multiple application frameworks and runtimes.
+
+{PRODUCT} is build on open standards (DMN, XML, JSON, JAX-RS) and de-facto standards (Git, Maven, Eclipse, IntelliJ).
+
+{PRODUCT} is supported by Red Hat and is an equivalent of the open-source community project, Drools. {PRODUCT} has the following capabilities:
```

Comment:
The sentence "Red Hat Decision Manager is supported by Red Hat and is an equivalent of the open-source community project, Drools." is not required. 
You can start this paragraph with  "Red Hat Decision Manager has the following capabilities" directly.

Reply:
Changed as suggested.